src/ServerAgent.cpp

```diff
@@ -153,8 +156,7 @@ void HTTPServerAgent::ThreadMain()
 		// done with the queue
 		SDL_UnlockMutex(m_requestQueueLock);
 
-		Json::FastWriter writer;
-		req.buffer = writer.write(req.data);
+		req.buffer = req.data;
```

Comment:
@sturnclaw this is one of the two problems I could do with some advice about, this line is not an equivalent replacement for the `writer.write(req.data)` which used to be there which I believe serialised the data.